routes/services/study/updateStudy.ts

```
@@ -0,0 +1,13 @@
+import prisma from "../../prisma";
+import { RequestHandler } from "express";
+import * as s from "superstruct";
+import { PatchStudy } from "../../../struct";
+
+const updateStudy: RequestHandler = async (req, res, next) => {
+  try {
+  } catch (error) {
+    next(error);
+  }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)